contributed_definitions/nyaml/NXsts.yaml

```diff
@@ -177,6 +163,13 @@ NXsts(NXsensor_scan):
       # doc: Select whether to record the Z position during Z averaging time at the end of the
       # sweep (after Z control time) and store the average value in the header of the file
       # when saving. Using this option increases the sweep time by Z averaging time. (e.g. FALSE)
+      # run:
```

Comment:
What is the reason to have all of this commented out? If this is TODO, it should be labelled as such.

In general, if we want to propose this to NIAC in September, all of these open things should be resolved as much as possible.

Reply:
The concept `run` is redundant. We have `status` which say `lock-in` is running or not.